pkgs/applications/graphics/weylus/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, dbus
+, stdenv
+, ffmpeg
+, gst_all_1
+, xorg
+, libdrm
+, libva
+, fetchzip
+, copyDesktopItems
+, fontconfig
+, libpng
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "weylus";
+  version = "0.11.2";
+
+  src = fetchzip {
+    url = "https://github.com/H-M-H/Weylus/releases/download/v${version}/linux.zip";
```

Comment:
Why are we not building from source? Rust shouldn't be to hard.

Reply:
It's explained in the PR description:

> It's not a binary source package due to the build process being very complex, having to compile ffmpeg and headers for graphics from source separately. I tried packaging it source based but it ended up looking something like this, and even this failed to build for no reason that I could understand: https://github.com/legendofmiracles/nixpkgs/commit/3689ae97319418fd56aba0f16677f5fcf27ae27b

It needs typescript, C++, C and uses different build flags for ffmpeg, x264, nv-headers and libva, it expects to compile them itself.

I also had to do a lot of patching.

In short, it's way to complex and would only waste time on hydra.